derp/derp_server.go

```diff
@@ -1514,7 +1514,7 @@ func (c *sclient) sendLoop(ctx context.Context) error {
 		}
 	}()
 
-	jitter := time.Duration(rand.Intn(5000)) * time.Millisecond
+	jitter := rand.N(5 * time.Second)
```

Comment:
Not that this matters, but technically this is different; we'd previously sleep between 1-5000ms, rounded to the millisecond, now we jitter in nanoseconds

Reply:
yeah, this was easier to read though :) 
and as you said, doesn't really matter and don't have to do conversions back and forth.